apis/fluentd/v1alpha1/tests/expected/fluentd-namespaced-cfg-output-es.cfg

```diff
@@ -7,17 +7,17 @@
   @id  main
   @type  label_router
   <route>
-    @label  @4a579b91a4b1f32966ef08e305396a08
+    @label  @2d9e59757d3bfc66d93c3bc44b408922
     <match>
-      namespaces  kubesphere-logging-system
+      namespaces  fluent
     </match>
   </route>
 </match>
-<label @4a579b91a4b1f32966ef08e305396a08>
+<label @2d9e59757d3bfc66d93c3bc44b408922>
   <match **>
-    @id  FluentdConfig-kubesphere-logging-system-fluentd-config::cluster::clusteroutput::fluentd-output-es-0
+    @id  FluentdConfig-fluent-fluentd-config::cluster::clusteroutput::fluentd-output-es-0
     @type  elasticsearch
-    host  elasticsearch-logging-data.kubesphere-logging-system.svc
+    host  elasticsearch-logging-data.fluent.svc
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc